application/src/test/java/org/opentripplanner/updater/trip/siri/moduletests/IncompleteMessageTest.java

```diff
@@ -0,0 +1,41 @@
+package org.opentripplanner.updater.trip.siri.moduletests;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.opentripplanner.updater.spi.UpdateError.UpdateErrorType.TOO_FEW_STOPS;
+
+import java.util.Set;
+import org.junit.jupiter.api.Test;
+import org.opentripplanner.updater.trip.RealtimeTestConstants;
+import org.opentripplanner.updater.trip.RealtimeTestEnvironment;
+import org.opentripplanner.updater.trip.TripInput;
+import org.opentripplanner.updater.trip.siri.SiriEtBuilder;
+
+class IncompleteMessageTest implements RealtimeTestConstants {
```

Comment:
Back when I started adding these module tests @t2gran asked me to put these into packages to group them by feature. If you like this pattern, can you put this one into the package `rejection`?

Reply:
Done